.circleci/config.yml

```diff
@@ -80,14 +75,10 @@ jobs:
     executor: node8
     steps:
       - run-lint
-  cocoa-pods:
+  unit-tests:
     executor: noderuby
```

Comment:
Do we know which version of Node is run on this image? Would be cool if it was the latest or so

Reply:
I couldn’t spot an explicit listing of the version in the CI output, but it’s using `circleci/ruby:2.4-node` and in [this listing](https://hub.docker.com/layers/circleci/ruby/2.4-node/images/sha256-fb869ba73c3c712621031d17a28ebb6f38b998ec22efdeec354b870b50c1bd58?context=explore) I see `ENV NODE_VERSION=12.16.2`. So probably that?